.github/PULL_REQUEST_TEMPLATE/accept-reject-pep.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Accept/Reject PEP
+about: Mark a draft PEP as accepted or rejected
+---
+
+<!--
+
+See PEP 1 for more details about the PEP review and resolution process:
+
+https://peps.python.org/pep-0001/#pep-review-resolution
+
+Make sure to include the PEP number in the pull request title; for example:
+
+PEP NNN: Mark as Accepted/Rejected
+
+Thanks!
+
+-->
+
+
+# Accept/Reject PEP
+
+<!--
+
+Link to the official acceptance/rejection message
+by the Steering Council/PEP Delegate on the PEP's discussion thread.
+
+-->
+
+
+
+## Checklist
+
+<!--
+
+This lists the key steps needed to mark a PEP as Accepted/Rejected.
+
+You can help complete it yourself if you like
+by ticking any boxes you're sure about, like this: [x]
+
+If you're unsure about anything, just leave it blank and we'll take a look.
+
+-->
+
+
+* [ ] SC/PEP Delegate has formally accepted/rejected the PEP and posted to the ``Discussions-To`` thread
+* [ ] ``Status`` changed to ``Accepted``/``Rejected``
+* [ ] ``Resolution`` link points directly to SC/PEP Delegate official acceptance/rejected post
+* [ ] Acceptance/rejection notice added, if the SC/PEP delegate had any major conditions or critical commentary (and any changes made accordingly)
+* [ ] ``Discussions-To``, ``Post-History` and ``Python-Version`` complete and up to date
```

Comment:
Missing backtick here (although single backticks are more usual for Markdown (unlike reST))

```suggestion
* [ ] ``Discussions-To``, ``Post-History`` and ``Python-Version`` complete and up to date
```

Reply:
Implemented in Petr's suggestion, thanks!